.github/workflows/tests.yml

```diff
@@ -15,12 +15,7 @@ concurrency:
 
 jobs:
   authorize:
-    environment: ${{
-      github.event_name == 'pull_request_target' &&
-      github.event.pull_request.head.repo.full_name != github.repository
-      && 'external'
-      || 'internal'
-      }}
+    environment: ${{ github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.full_name != github.repository && 'external' || 'internal' }}
```

Comment:
I strongly disagree with `pre-commit` on this one: 3ce0369e09f83b71c332830fce6bd9e0ca4ca90d 🫠